### ISSUE TITLE
Training batches for imbalanced datasets

### ISSUE THREAD

**Issue by antriksh63** [dennybritz/cnn-text-classification-tf#151]
I have an imbalanced data set with around 100 entries of positive class and 4000 entries of negative class. One way to create the training batches would be to take 100 entries of positive and 100 entries of negative class and then allow the code to proceed as normal. However, this has high chances of overfitting.
I think one thing that can be done is to have equal number of entries of positive and negative entries in the training batch.( batch_size/2 positive and negative entries). How can I do this?

**Comment by self-ms** [comment]
If your data is embedded and labels are available, you can use the following repository:
https://github.com/ms-unlimit/Transformer-Based-Machine-Learning-Framework

**Comment by xuqiangxq** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by liangtingStduy** [comment]
This is an automatic reply,confirming that your e-mail was received.Thank you!-------------------------------------liangting
